fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/package-info.java

```diff
@@ -1,6 +1,10 @@
 /**
- * Classes related to the UI and logic of the LAO detail view The needed activities, fragments,
- * adapters and listeners are inside this package. One view model is present for the logic of the
- * LAO detail view.
+ * The package detail represents the LAO-specific view.
+ * <p>
+ * The package contains one activity, one view model and the necessary adapters and listeners for the view.
+ * <p>
```

Comment:
is it a `<p>` as in HTML (with a missing closing tag?), or is there a specific syntax here?